### PR TITLE
add 64x32 oled support

### PR DESCRIPTION
**Pull request by wemos**
add 0.49inch 64x32  oled support

### CODE REVIEW

**Comment by marcelstoer** [comment]
Thanks for your contribution. Unfortunately we haven't got any such module to verify the correctness of the code ourselves. Could you maybe add a few photos or a video of your test setup?

**Comment by wemos** [comment]
I upload a video
https://youtu.be/XfhqUSqeIRc

File: src/OLEDDisplay.h [hunk 1]
```diff
@@ -139,7 +139,8 @@ enum OLEDDISPLAY_GEOMETRY {
   GEOMETRY_128_64   = 0,
   GEOMETRY_128_32   = 1,
   GEOMETRY_64_48    = 2,
-  GEOMETRY_RAWMODE  = 4
+  GEOMETRY_RAWMODE  = 4,
+  GEOMETRY_64_32    = 5
```
Comment (marcelstoer) [inline src/OLEDDisplay.h:143]: Hmm, shouldn't this be placed _before_ the raw mode and id 3 instead of 5?

**Comment by marcelstoer** [comment]
Thanks for the video!

File: src/OLEDDisplay.h (hunk 1 above)
Comment (wemos) [reply src/OLEDDisplay.h]: ok, I changed this, please check.

**Comment by wemos** [comment]
I changed that, please check.

**Pull request merged**